native/src/hooks/useLoadResourceCache.ts

```diff
@@ -0,0 +1,97 @@
+import NetInfo from '@react-native-community/netinfo'
+import { flatten, mapValues, pickBy, reduce, values } from 'lodash'
+import moment from 'moment'
+
+import { CategoriesMapModel, EventModel, ExtendedPageModel, PoiModel } from 'api-client'
+
+import buildConfig from '../constants/buildConfig'
+import { PageResourceCacheEntryStateType } from '../redux/StateType'
+import dataContainer from '../utils/DefaultDataContainer'
+import FetcherModule, { TargetFilePathsType } from '../utils/FetcherModule'
+import ResourceURLFinder from '../utils/ResourceURLFinder'
+import buildResourceFilePath from '../utils/buildResourceFilePath'
+
+type FetchMapTargetType = {
+  url: string
+  filePath: string
+  urlHash: string
+}
+
+type UseLoadResourceCacheProps = {
+  cityCode: string
+  languageCode: string
+  categories: CategoriesMapModel | null
+  events: EventModel[] | null
+  pois: PoiModel[] | null
+}
+
+const loadResourceCache = async ({
```

Comment:
Mostly just copied from `fetchResourceCache` and `loadCityContent` sagas, added loading of poi resources